JustPeek/Classes/PeekNativeHandler.swift

```diff
@@ -21,6 +23,11 @@ internal class PeekNativeHandler: NSObject, PeekHandler, UIViewControllerPreview
     
     func previewingContext(_ context: UIViewControllerPreviewing, viewControllerForLocation l: CGPoint) -> UIViewController? {
         peekContext?.destinationViewController = delegate?.peekContext(peekContext!, viewControllerForPeekingAt: l)
+        
+        if peekContext?.destinationViewController != nil {
+            let gesture = context.previewingGestureRecognizerForFailureRelationship
+            gesture.addObserver(self, forKeyPath: ObservingKeyPath, options: .new, context: nil)
```

Comment:
In my case PeekNativeHandler's peekContext.destinationViewController should be nil when UIGestureRecognizerState is ended. It wasn't nil until another peek event is occured.